netbox_agent/network.py

```diff
@@ -60,7 +60,19 @@ def scan(self):
                 continue
 
             ip_addr = netifaces.ifaddresses(interface).get(netifaces.AF_INET)
-            ip_addr += netifaces.ifaddresses(interface).get(netifaces.AF_INET6)
+            ip6_addr += netifaces.ifaddresses(interface).get(netifaces.AF_INET6)
+
+            # netifaces returns a ipv6 netmask that netaddr does not understand.
+            # this strips the netmask down to the correct format for netaddr.
+            # ie, this:
+            #   {'addr': 'fe80::ec4:7aff:fe59:ec4a%eno1.50', 'netmask': 'ffff:ffff:ffff:ffff::/64'}
+            # becomes:
+            #   {'addr': 'fe80::ec4:7aff:fe59:ec4a%eno1.50', 'netmask': 'ffff:ffff:ffff:ffff::'}
+            #
+            for addr in ip6_addr:
+                addr["netmask"] = re.sub("/\d+$", "", addr["netmask"])
```

Comment:
I've tried to reproduce this case during my testing, but it was unsuccessful. It's goot you caught this!
We should do the same thing for `addr` and remove `%.*` no ?